### ISSUE TITLE
ACC-263.1 (Level A) _ Modify Modules _ Text _ No heading level provided for a heading

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3139]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Text
Affected Check Point:	ACC-263.1
Short text:	ACC-263.1 (Level A) _ Modify Modules _ Text _ No heading level provided for a heading

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Press ‘Insert+F6’ key to see the list of heading levels in the page.

3. Issue Type: Manual
Observed Behavior:
The main header text ‘Modules’ within ‘Modules’ page is not provided a heading level even though it is a heading. 
Screenshot:
 

Expected Behavior:
Correct heading should be defined for all the pages in the application. Users should order heading elements properly. The heading element level must match the level communicated visually by the headings on the page. Headings can be used to communicate structure as well as important content although their purpose should generally be to communicate the structure of content. 

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.

________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-263.1 (Level A) _ Modify Modules _ Text _ No heading level provided for a heading.docx](https://github.com/user-attachments/files/16659861/ACC-263.1.Level.A._.Modify.Modules._.Text._.No.heading.level.provided.for.a.heading.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659862/Issue.processing-09.2021.002.docx)

**Comment by OliwiaGowor** [comment]
It's connected to this [issue](https://github.com/SAP/ui5-webcomponents/issues/10747) - we will have to wait for UI5 React 2.7 release.
